Source/interactivity_manager_impl.cpp

```diff
@@ -1167,6 +1187,10 @@ interactivity_manager_impl::on_socket_message_received(
     _In_ const string_t& message
 )
 {
+    // We always return all messages from the service to to the developer, even the ones the SDK will process them.
+    std::shared_ptr<interactive_message_event_args> args = std::shared_ptr<interactive_message_event_args>(new interactive_message_event_args(message));
+    queue_interactive_event_for_client(L"", std::error_code(0, std::generic_category()), interactive_event_type::unknown, args);

```

Comment:
What I'm saying is that it's a ton of extra events for developers to have to deal with. They're getting raw JSON for button presses as well as events with objects? So now they're paying the cost for twice as many events as they would have before, for every message. Just seems super verbose.

---
In reply to: [132452652](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132452652) [](ancestors = 132452652,132343459,132343213)